.github/workflows/build-custom-docker-image.yml

```diff
@@ -0,0 +1,38 @@
+name: OpenSearch Dashboards Build Custom Docker Image
+
+on:
+  workflow_dispatch:
+
+jobs:
+  Build-Custom-Docker-Image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
use checkout@v3 instead of v2.
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout, actions/checkout

Reply:
Awesome, good to know! Will do, thx

Reply:
Changed all workflow steps to use checkout@v3 in [this commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/afa1c7d57264e293228d81b98676ddfc36f9cac4)